app/models/admin_ability.rb

```diff
@@ -292,6 +292,8 @@ def signed_in_with_track_organizer_role(user)
       role.resource_type == 'Conference' || role.resource_type == 'Track'
     end
 
+    cannot :toggle_user, Role
+
     can :toggle_user, Role do |role|
       role.resource_type == 'Track' && track_ids_for_track_organizer.include?(role.resource_id)
```

Comment:
The permissions you wan is exactly that here, right? Why do you need the `cannot`?

Reply:
You are right, I just realized what I stumbled upon was an organization admin being able to make all these fancy changes! Thanks!